app/test/testmerginapi.cpp

```diff
@@ -805,9 +805,11 @@ void TestMerginApi::testConflictRemoteUpdateLocalUpdate()
   uploadRemoteProject( mApi, mUsername, projectName );
 
   // verify the result: the server version should be in test1.txt
-  // and the local version should go to test1.txt_conflict
+  // and the local version should go to test1.txt_conflict_<username>_<version>
+  LocalProjectInfo info = mApi->localProjectsManager().projectFromDirectory( projectDir );
+  QString conflictFilename = filename + "_conflict_" + mUsername + "_v" + info.localVersion;
```

Comment:
Better to use fixed expected version in the tests rather than querying it from the local project info: e.g.
`... + mUsername + "_v123";`